### PR TITLE
SISRP-15443 - When viewing as a delegate, the gear/popover image is suppressed entirely

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-15443

### CODE REVIEW

**Comment by christianvuerings** [comment]
@johncrossman would it be ok for you to review?

File: src/assets/templates/widgets/gear_popover.html [hunk 1]
```diff
@@ -4,9 +4,9 @@
   data-ng-class="{true:'cc-header-icon-selected'}[api.popover.status('cc-popover-menu')]"
   title="Settings">
     <div class="cc-flex cc-flex-align-center">
-      <span data-ng-if="api.user.profile.hasPhoto" class="cc-popover-gear-round-profile cc-popover-gear-round-profile-has-photo">
+      <span data-ng-if="api.user.profile.hasPhoto && !api.user.profile.delegateActingAsUid" class="cc-popover-gear-round-profile cc-popover-gear-round-profile-has-photo">
       </span>
-      <span data-ng-if="!api.user.profile.hasPhoto" class="cc-popover-gear-round-profile cc-popover-gear-round-profile-no-photo">
+      <span data-ng-if="!api.user.profile.hasPhoto || api.user.profile.delegateActingAsUid" class="cc-popover-gear-round-profile cc-popover-gear-round-profile-no-photo">
```
Comment (johncrossman) [inline src/assets/templates/widgets/gear_popover.html]: @dfkaye I'd prefer we put this logic in `User::Api` class:
```
hasPhoto: User::Photo.has_photo?(@uid) && !authentication_state.original_delegate_user_id,
```

Comment (johncrossman) [inline src/assets/templates/widgets/gear_popover.html]: **Note:** `authentication_state.original_delegate_user_id` indicates a delegate user in view-as mode.

**Comment by johncrossman** [comment]
Nice. Thank you, @dfkaye

**Pull request merged**